pubtools/_quay/signature_handler.py

```diff
@@ -536,21 +538,24 @@ def construct_index_image_claim_messages(self, index_image, tag, signing_keys):
                     continue
                 for digest in digests:
                     repo = self.target_settings["quay_operator_repository"]
-                    reference = image_schema.format(host=registry, repository=repo, tag=tag)
-                    claim_message = self.create_manifest_claim_message(
-                        destination_repo=repo,
-                        signature_key=signing_key,
-                        manifest_digest=digest,
-                        docker_reference=reference,
-                        image_name=repo,
-                        task_id=self.task_id,
-                    )
-                    claim_messages.append(claim_message)
+                    for _tag in (tag, stamp_tag):
+                        if not _tag:
+                            continue
+                        reference = image_schema.format(host=registry, repository=repo, tag=_tag)
+                        claim_message = self.create_manifest_claim_message(
+                            destination_repo=repo,
+                            signature_key=signing_key,
+                            manifest_digest=digest,
+                            docker_reference=reference,
+                            image_name=repo,
+                            task_id=self.task_id,
+                        )
+                        claim_messages.append(claim_message)
 
         return claim_messages
 
     @log_step("Sign operator images")
-    def sign_operator_images(self, iib_results):
+    def sign_operator_images(self, iib_results, index_stamp):
```

Comment:
missing docstring

Reply:
Also, I would probably rename this variable. The fact that it's a timestamp is not relevant here, only that it's a redundant tag